org-chart/src/contexts/AvatarContext.tsx

```diff
@@ -0,0 +1,105 @@
+        { id:"10", name: 'Korra', skill: 'Water Bender', imgSrc: 'https://decider.com/wp-content/uploads/2020/08/the-legend-of-korra.jpg?quality=75&strip=all'},
+        { id:"11", name: 'Bolin', skill: 'Earth Bender', imgSrc: 'https://a1cf74336522e87f135f-2f21ace9a6cf0052456644b80fa06d4f.ssl.cf2.rackcdn.com/images/characters/large/800/Bolin-The.Legend.of.Korra.webp'},
+        { id:"12", name: 'Mako', skill: 'Fire Bender', imgSrc: 'https://upload.wikimedia.org/wikipedia/en/3/35/Mako_%28The_Legend_of_Korra%29.jpg'}
+    ],
+    'team_kyoshi': [
+        { id:"1", name: 'Kyoshi', skill: 'Earth Bender', nextAvatarTeam: 'team_roku', imgSrc: 'https://img.buzzfeed.com/buzzfeed-static/static/2020-08/13/6/asset/f0d82f7856c0/sub-buzz-3521-1597299837-1.png'},
+    ],
+}
+
+const AVATAR_SEQUENCE = ["team_kyoshi", "team_roku", "team_aang", "team_korra"]
+
+
+type AvatarContextType = {
+    getAvatarTeam: (teamName: string) => Member[],
+    visibleTeams: Set<string>,
+    setVisibleTeams: Dispatch<SetStateAction<Set<string>>>,
+    isTheLastAvatar: (name: string) => boolean,
+    updateVisibleTeams: (name: string) => void
+}
+
+export const AvatarContext = createContext<AvatarContextType>({
+    getAvatarTeam: (teamName: string) => TEAMS[STARTING_TEAM_NAME],
+    visibleTeams: new Set(),
+    setVisibleTeams: (prevState: SetStateAction<Set<string>>) => prevState,
+    isTheLastAvatar: (name: string) => false,
+    updateVisibleTeams: (name: string) => undefined
+})
+
+
+export function AvatarProvider({children}: any) {
+    const [visibleTeams, setVisibleTeams] = useState<Set<string>>(new Set([STARTING_TEAM_NAME])) 
+
+    function getAvatarTeam(teamName: string): Member[] {
+        return TEAMS[teamName]
+    }
+
+    function isTheLastAvatar(name: string): boolean {
+        const teamName = "team_" + name.toLowerCase()
+        return ( Array.from(visibleTeams).pop()  === teamName)
+    }
+
+    function updateVisibleTeams(teamName: string){
+          // find the teams that are in the visible teams but are not supposed to be since they are 
+            // after the last actually visible team
+            let teamIndex = AVATAR_SEQUENCE.length + 1
+            let updatedTeams = new Set(visibleTeams)
+            for (let i = 0; i < AVATAR_SEQUENCE.length; i++) {
+                if (AVATAR_SEQUENCE[i] === teamName) {
+                    teamIndex = i;
+                }
+                if (i > teamIndex) {
+                    updatedTeams.delete(AVATAR_SEQUENCE[i])
+                }
+            }
+            if (updatedTeams !== visibleTeams) {
+               setVisibleTeams(updatedTeams)
+            }
+    }
+
+
+    const defaultValue = {
+        getAvatarTeam: getAvatarTeam,
+        visibleTeams: visibleTeams,
+        setVisibleTeams: setVisibleTeams,
+        isTheLastAvatar: isTheLastAvatar,
+        updateVisibleTeams: updateVisibleTeams
+    }
```

Comment:
```suggestion
    const defaultValue = {
        getAvatarTeam,
        visibleTeams,
        setVisibleTeams,
        isTheLastAvatar,
        updateVisibleTeams
    }
```

Unless you're renaming them, you don't need to repeat the `key` and the `value` in JS when defining object properties